docs/source/how-to/deploying-substra/upgrade-notes.rst

```diff
@@ -4,6 +4,93 @@
 Upgrade notes
 *************
 
+.. _ops upgrade notes 0.34:
+
+Substra 0.34.0
+--------------
+
+This version upgrades the integrated PostgreSQL databases from versions 14 to 16. This upgrade must be performed manually.
+
+If you :ref:`do not use the integrated databases <ops howto external database>`, then you have nothing to do.
+
+Otherwise, you must update the data format. There are many ways to do this; here we give you a simple way which is acceptable if you have small amounts of data, you have access to the clusters, and downtime is acceptable. It must be repeated on each Substra node and orchestrator.
+
+#. Make sure no operations are in progress and cordon the node.
+
+#. Gather required info:
+   * database credentials (here ``$PG_USER`` and an associated password)
+   * Kubernetes namespace (here ``$NAMESPACE``)
+   * Helm release (hereafter ``$RELEASE_NAME``, can be obtained from ``helm get release -A``)
+   * hostname of the Postgres service (hereafter ``$HOST``, can be obtained from ``kubectl get svc -n $NAMESPACE``)
+   * name of the database in Postgres: by default ``orchestrator`` for the orchestrator, and ``substra`` for the backend (hereafter ``$DB``)
```

Comment:
Not sure that is the expected render:

![image](https://github.com/Substra/substra-documentation/assets/135698225/549f2499-970e-44f8-879d-42356753fcd6)

---

```suggestion
   - database credentials (here ``$PG_USER`` and an associated password)
   - Kubernetes namespace (here ``$NAMESPACE``)
   - Helm release (hereafter ``$RELEASE_NAME``, can be obtained from ``helm get release -A``)
   - hostname of the Postgres service (hereafter ``$HOST``, can be obtained from ``kubectl get svc -n $NAMESPACE``)
   - name of the database in Postgres: by default ``orchestrator`` for the orchestrator, and ``substra`` for the backend (hereafter ``$DB``)
```

Reply:
Same thing in rst no ?

Reply:
good catch. I think its a blanck line missing